### PR TITLE
Deletes Autolathe

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
Deletes the autolathe. Moves most things that they can print to more appropriate vendors.
image
image
image

Closes https://github.com/tgstation/TerraGov-Marine-Corps/issues/12056

## Why It's Good For The Game
This old machine has no place in TGMC anymore to be honest. It's single-handedly responsible for the reason why we can't just vend req boxes from vending machines. 
Why are there so many bandaid fixes on "cable coil makes infinite metal" "Surgical tools produce infinite metal" "Bla bla bla infinite metal".
This machine is old and unnecessary in TGMC. It's time to let go, delete it from the game.

## Changelog

:cl:
add: Crates can be vended from the Automated Weapons rack in the Boxes section (They don't give metal when welded)
add: Conveyor belts and switches are in the req vendor under Reqtorio Basics.
add: Beakers to Nanotrasen Med Plus
add: Various engineering thingies to engi-vend.
add: Welding goggles and helmets to engi-vend.
add: Table parts, reinforced table parts and rack parts can be made from metal sheets in-hand.
add: You can weld 4 floor tiles to make 1 metal.
del: Removes the autolathe
/:cl:

### CODE REVIEW

**Comment by lKiseki** [comment]
you-dropped-this-king-you-dropped-this-queen

**Comment by Tailiik** [comment]
As long as I can get muh beakers still

**Comment by FosterGrime-Heymynameslouie** [comment]
we be cool if the items that the autolathe prints be moved into other vendors, or an entirely new vendor, something like "utility vendor"

metal tiles, flashlights, handheld radios, buckets, beakers and medical equipment is still needed from time to time and completely removing them from being accessed is pretty meh

**Comment by Xander3359** [comment]
> we be cool if the items that the autolathe prints be moved into other vendors, or an entirely new vendor, something like "utility vendor"

Thats the plan, and the reason why this is in draft, I'm going through the list and looking at all the things that I need to move to vendors, and going to move them to vendors I deem the most appropriate (open to feedback)

**Comment by Three333Digit** [comment]
Small and large beakers needs to be infinitely available in the medvendor.

**Comment by chizzy376** [comment]
Horrible PR, shipside was made to be scrapped. If you allow to make free tables from vendor, you'll get infinite metal by disassembling them. And you can't just make all tables not give metal by being disassembled, as that heavily nerfs engineer scavenging yet again. If you don't allow to make tables from vendor, we will never be able to create them and every destroyed table can never be replaced. Both options are bad. And this is an example just for tables, imagine the 50+ other things you're removing as well. Rack parts, conveyors, conveyor switches, etc. Please do not merge this.

**Comment by Xander3359** [comment]
> imagine the 50+ other things you're removing as well. Rack parts, conveyors, conveyor switches, etc. Please do not merge this.

Conveyors and conveyor switches are in the vendor. Most things have been moved to vendors. (See screenshots)
Have you seriously ever built tables? I can move them to the vendor if you'd like, but yeah they would have to be the variants that dont give metal (This kind already exists)
Alternatively, using metal in hand to craft rack parts, table frame, etc.

**Comment by Mrrpip** [comment]
> > imagine the 50+ other things you're removing as well. Rack parts, conveyors, conveyor switches, etc. Please do not merge this.
> 
> Conveyors and conveyor switches are in the vendor. Most things have been moved to vendors. (See screenshots) Have you seriously ever built tables? I can move them to the vendor if you'd like, but yeah they would have to be the variants that dont give metal (This kind already exists) Alternatively, using metal in hand to craft rack parts, table frame, etc.

On the other hand though, With infinite tables, you could infinitely flip them for makeshift cades, even if they do not grant metal. So do keep that in mind.

**Comment by Xander3359** [comment]
You can wrench a table and get metal from it, thus, I've added it so that 1 metal can make 1 table. (Same applies for rack parts and reinforced table frame)

**Comment by FosterGrime-Heymynameslouie** [comment]
> > we be cool if the items that the autolathe prints be moved into other vendors, or an entirely new vendor, something like "utility vendor"
> 
> Thats the plan, and the reason why this is in draft, I'm going through the list and looking at all the things that I need to move to vendors, and going to move them to vendors I deem the most appropriate (open to feedback)

BALLER

you have my full support cool pr

**Review by RipGrayson — APPROVED** [review]
Mapping part is approved, up to other maintainers on the balance/concept side of this.

**Comment by Losenis** [comment]
If you actually make tables and other dense objects craftable from metal, give them a LENGTHY timer to build because table is instant to place and can actually get xenos killed like a crusher charging in and then literally instantly getting blocked by like 20 tables that even if are one hit to destroy (pretty sure they actually aren't), it'd still have enough of an impact to be a balance concern once it's used enough.

File: code/game/objects/items/stacks/sheets/sheet_types.dm
Comment (Lumipharon) [inline code/game/objects/items/stacks/sheets/sheet_types.dm:30]: critical balance issue: rtable same cost as normal table

File: code/game/objects/items/stacks/tiles/plasteel.dm
Comment (Lumipharon) [inline code/game/objects/items/stacks/tiles/plasteel.dm:24]: ??

File: code/game/objects/machinery/vending/marine_vending.dm
Comment (Lumipharon) [inline code/game/objects/machinery/vending/marine_vending.dm:883]: this is the real vendor, not the valhalla one

File: code/__DEFINES/machines.dm
Comment (Lumipharon) [inline code/__DEFINES/machines.dm:33]: you can't just remove the line, adjust the bits.

**Review by Lumipharon — CHANGES_REQUESTED** [review]

File: code/game/objects/items/stacks/tiles/plasteel.dm
Comment (Xander3359) [reply code/game/objects/items/stacks/tiles/plasteel.dm:24]: Floor tiles.

File: code/game/objects/machinery/vending/marine_vending.dm
Comment (Xander3359) [reply code/game/objects/machinery/vending/marine_vending.dm:883]: image
Is 10 of each ok?

**Review by RipGrayson — APPROVED** [review]

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**
